Assets/MixedRealityToolkit/_Core/Interfaces/IMixedRealityDevice.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
+{
+    /// <summary>
+    /// Mixed Reality Toolkit device definition, used to instantiate and manage a specific device / SDK
+    /// </summary>
+    public interface IMixedRealityDevice
```

Comment:
Not sure if this is needed. Windows doesn't give us specific devices to choose from, but only controllers, gamepads, and their data.

Reply:
It's the generic device interface for ALL SDK's not just Windows